aeolis/tests/test_model.py

```diff
@@ -100,8 +100,8 @@ class TestAeoLiSRunner:
 
     def test_parse_callback_from_module(self):
         """Test if the callback function can be loaded from a file"""
-        runner = AeoLiSRunner("aeolis/tests/aeolis.txt")
-        callback = runner.parse_callback("aeolis/tests/callback_example.py:mock_callback")
+        runner = AeoLiSRunner("aeolis.txt")
+        callback = runner.parse_callback("callback_example.py:mock_callback")
```

Comment:
This path to the callback function is incorrect. The correct path is "aeolis/tests/callback_example.py:mock_callback"